### PR TITLE
Added documentation link in readme

### PR DESCRIPTION
**Pull request by sanjanag**
----
:books: Documentation preview :books:: https://cleanvision--96.org.readthedocs.build/en/96/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +34,11 @@ imagelab.find_issues(issue_types)
...
 ```
 
-More on how to get started with CleanVision: 
+More on how to get started with CleanVision:
 
-- [example script](https://github.com/cleanlab/cleanvision/blob/main/examples/run.py)
-- [jupyter notebook demo](https://github.com/cleanlab/cleanvision/blob/main/examples/demo.ipynb)
+- [Example script](https://github.com/cleanlab/cleanvision/blob/main/examples/run.py)
+- [Jupyter notebook tutorial](https://github.com/cleanlab/cleanvision/blob/main/examples/demo.ipynb)
+- [Documentation](https://cleanvision.readthedocs.io/en/latest/)
```
Comment (jwmueller) [inline README.md:41]:
```suggestion
- [Documentation](https://cleanvision.readthedocs.io/)
```

Comment (jwmueller) [reply README.md:41]: seems like this shorter one should be the official link?

Comment (sanjanag) [reply README.md]: shorter one redirects to latest, I think either of them works

Comment (jwmueller) [reply README.md]: I am saying we should have one official link we consistently share everywhere, not multiple, and it should be the most concise one

**Review by jwmueller — APPROVED** [review]

**Pull request merged**